package/src/main/java/com/microsoft/azure/cosmos/cassandra/CosmosFailoverAwareRRPolicy.java

```diff
@@ -42,6 +42,7 @@
  * The nodes in the default write region are retrieved from the global endpoint
  * of the account, based on the dns refresh interval of 60 seconds by default.
  */
+@Deprecated
```

Comment:
<p> Add a line saying that this class is deprecated, use CosmosLoadBalancingPolicy class